backend/shared/createTransactionViewModel.js

```diff
@@ -5,13 +5,15 @@ module.exports = (transactions, prisons, showBalance = true, mergeAmounts = fals
     const { description: prisonName } = prisons.find(agency => transaction.agencyId === agency.agencyId)
     const { penceAmount, currentBalance } = transaction
 
+    const amount = (typeof penceAmount === 'number' && penceAmount / 100) || ''
```

Comment:
can't this just be `formatCurrency(penceAmount)` then you won't need the `formatCurrency` below

Reply:
Agreed a lot of duplication here. I'll clean it up.